src/main/java/uk/gov/register/core/Field.java

```diff
@@ -47,16 +51,28 @@ public String getText() {
     }
 
     @Override
-    public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
+    public boolean equals(final Object o) {
+        if (this == o) {
```

Comment:
We don't do one-liners in any other places (getters, setters are common suspects), why do it here?

In fact, now that we are using the common-lang3 we can use the HashCodeBuilder and the EqualsBuilder that help a lot with readability.

Reply:
It's true. I think the reason we do one-liners here (and I may be wrong about this) is because we have a lot of individual checks which are small enough that they can be one-liners, and that by doing this we keep the code compact (albeit slightly harder to read).

I'm generally in favour of not having one-liners, but you have to read through a lot more of the code to get to the end in this example. The one-liners also give me the impression at least that the method is doing some fairly simple checks, as they aren't complex enough to need to span multiple lines, and makes it easier to get a sense of the flow of success or failure of the method.

Reply:
I just think we should use the same format for everything not just depending of it is simple all not, specially, because this 'simple' classification can be very subjective. But, we can discuss this us a team. Anyway, I'm going to disconnect the auto-format and save actions.